specification/grammars/annotation.lark

```diff
@@ -38,6 +38,14 @@ NUMBER : DIGIT ("." (DIGIT)+)?
 
 AMINO_ACID : "A" | "R" | "N" | "D" | "C" | "E" | "Q" | "G" | "H" | "K" | "M" | "F" | "P" | "S" | "T" | "W" | "Y" | "V" | "I" | "L"
```

Comment:
Is there a strong reason to limit ourselves to these? The next most common would be selenocysteine (U). Plenty of human proteins have selenocysteine, e.g.:
https://www.uniprot.org/uniprotkb/P07203/entry#sequences
UWPR also defines O:
https://proteomicsresource.washington.edu/protocols06/masses.php
J is defined as I or L:
https://www.bioinformatics.org/sms2/iupac.html
for which you can calculate an exact mass.
One could argue that the rest you can't calculate a mass for because they're ambiguous.
But still, why not allow them?
Oh, I think ProForma formally encourages X to have a 0 mass and you can do X[+123.0222] to specify some other artificial amino acid (of which there are plenty?) that might be common in synthetic peptides.

Why not just allow all A-Z?